### DISCUSSION TITLE
again modded minecraft crash when launching...

### DISCUSSION THREAD

**Discussion by KAARVAAH** [FabricMC/fabric-api#3424] [Player support] [answered]
### Minecraft Logs

```txt
https://mclo.gs/IDToyL8
```

### Description

modded minecraft crash when launching

**Comment by KAARVAAH** [comment]
someone can help me please? 
[latest.log](https://github.com/FabricMC/fabric/files/13357695/latest.log)

**Comment by Linguardium** [comment, marked as answer]
wrong version of nochatreports
